packages/yoastseo/src/languageProcessing/helpers/sanitize/sanitizeString.js

```diff
@@ -1,16 +1,17 @@
-/** @module stringProcessing/sanitizeString */
```

Comment:
I thought that it was safe to remove that since after the refactor project we moved many helpers to `languageProcessing/index.js` as opposed to `stringProcessing` in the past. The `sanitizeString` helper is also added to `languageProcessing` now and can be accessed this way:
````
import { languageProcessing } from "yoastseo";
const { sanitizeString } = languageProcessing;
````
But since maybe my understanding of module is not that strong, I can revert this if you think removing it not a safe thing to do.

Reply:
I think your reasoning is sound, but let's ping @manuelaugustin to be sure :-). What do you think, Manuel?

Reply:
As Aida explained, the documentation here is outdated because the functionality has moved. So let's remove it.

I think you're touching upon an interesting point though: we still have this `@module` documentation all over the place, mostly outdated. We're also not really actively adding it anymore. I think the documentation is still from the time when we were use node require/module.export and not yet ES6 modules. That being said, it seems like you can also [use this the module documentation with ES6 modules.](https://jsdoc.app/howto-es2015-modules.html). So it would be good to check with e.g. technical council if we should actually use this documentation, or if we otherwise should just remove it everywhere.

But in any case not a consideration for this issue 👍

Reply:
@hansjovis we think that the `@module` documentation topic above is an interesting point to bring up in the Technical council :) -- and also what should we do with our current outdated `@module` documentation.